indra/sources/tees/tees_api.py

```diff
@@ -104,6 +116,197 @@ def process_text(text, pmid=None, tees_path=None, python2_path=None):
                         'listed in the PATH environment variable. ' +
                         'Need python2 to run TEES.')
 
+    # Run TEES
+    a1_text, a2_text, sentence_segmentations = run_tees_on_text(text,
+                                                                tees_path,
+                                                                python2_path)
+
     # Run the TEES processor
-    tp = TEESProcessor(text, pmid, tees_path, python2_path)
+    tp = TEESProcessor(a1_text, a2_text, sentence_segmentations, pmid)
     return tp
+
+def run_tees_on_text(text, tees_path, python2_path):
+    """Runs TEES on the given text in a temporary directory and returns a
+    temporary directory with TEES output. The caller should delete this
+    directory when done with it.
+
+    Parameters
+    ----------
+    text: str
+        Text from which to extract relationships
+    tees_path: str
```

Comment:
Since there is a TEES_PATH config now, this argument is redundant, I think, and could be removed in favor of reading from the config.